Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -178,19 +178,19 @@ def puts_command
         puts "travis_fold:start:#{@travis_fold_id}"
         puts "travis_time:start:#{@travis_timer_id}"
       end
-      puts "#{Tty.blue}==>#{Tty.white} #{@command.join(" ")}#{Tty.reset}"
+      puts Formatter.headline(@command.join(" "), color: :blue)
```

Comment:
> I don't feel too strongly about this, but my impression is we want to avoid introducing the new hash syntax until we update our RuboCop configuration and make a sweeping change

This was done.